src/library_html5.js

```diff
@@ -105,7 +105,7 @@ var LibraryJSEvents = {
         var call = JSEvents.deferredCalls[i];
         JSEvents.deferredCalls.splice(i, 1);
         --i;
-        call.targetFunction.apply(this, call.argsList);
+        call.targetFunction.apply(null, call.argsList);
```

Comment:
Closure likes to perform "flattening" refactoring that can change the `this` object for a function, so it either wants to statically know what `this` refers to, so it will disable such flattening; or it gives a warning if it doesn't know.

In this case we don't need the `this` at all, these are all global functions that are called, so replace with `null` for simplicity.